src/components/backend-ai-session-launcher.ts

```diff
@@ -2959,11 +2960,10 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
   }
 
   _disableEnterKey() {
-    this.shadowRoot?.querySelectorAll('wl-expansion').forEach((element) => {
-      // TODO remove protected property assignment
-      (element as any).onKeyDown = (e) => {
-        const enterKey = 13;
-        if (e.keyCode === enterKey) {
+    this.shadowRoot?.querySelectorAll<Expansion>('wl-expansion').forEach((element) => {
+      // remove protected property assignment
+      (element as HTMLElement).onkeydown = (e) => {
```

Comment:
We can remove `as ..`
```suggestion
      element.onkeydown = (e) => {
```

Reply:
resolved.